### ISSUE TITLE
[EN][Profession][Alchemy][Quest] Double Alchemy Specialization

### ISSUE THREAD

**Issue by RelmAtlantiss** [Atlantiss/BugTracker#3041]
You can have two alchemy specializations at once.
This happens if you made your specialization during 4.0.6 patch (old quests).
You can do new alchemy quests from 4.3.4 patch without having to remove old specialization.
![alchemybug](https://cloud.githubusercontent.com/assets/11969548/7171500/95f89aa6-e3d9-11e4-9e97-4a2b9bef12a2.png)

Actual server revision: 4401

**Comment by Varchord** [comment]
Maybe it's visual. I've tried to replicate and failed ^^

**Comment by RelmAtlantiss** [comment]
I don't think it was just visual, didn't try to check because i didn't want to get banned (i removed old specialization). Anyways, it should be fixed.

**Comment by Varchord** [comment]
You wouldn't get banned for testing mate ^^

**Comment by RelmAtlantiss** [comment]
Using two specializations at once is exploiting, so i could get banned.

**Comment by Varchord** [comment]
*testing

I see it this way
You KNOW that it's showing 2 specializations
You DON'T KNOW if both specializations are working at once

**Comment by RelmAtlantiss** [comment]
I forget that i was changing faction recently. That might reset the specialization quest.

**Comment by wtRCH** [comment]
FACTION CHANGE? @BAndysc
